app/src/main/res/values/strings.xml

```diff
@@ -136,5 +136,7 @@
     <!--settings profile dialog-->
     <string name="message">Are you sure you want to log out?</string>
     <string name="cancel">Cancel</string>
+    <string name="no_liked_events">No Liked Events</string>
+    <string name="no_tickets">No Tickets</string>
```

Comment:
This string is not used anywhere right ?

Reply:
"no_tickets" string was supposed to be used to show "no tickets " in tickets fragment . But I couldn't fix it properly, I will try it after some time .